plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveErrorCode.java

```diff
@@ -68,6 +68,7 @@ public enum HiveErrorCode
     HIVE_VIEW_TRANSLATION_ERROR(41, EXTERNAL),
     HIVE_PARTITION_NOT_FOUND(42, USER_ERROR),
     HIVE_INVALID_TIMESTAMP_COERCION(43, EXTERNAL),
+    HIVE_UNSERIALIZABLE_JSON_VALUE(44, USER_ERROR),
```

Comment:
This is unused. Revert.

Reply:
`HiveFormatsErrorCode` has to match existing values in `HiveErrorCode` and must not conflict- so we need this value here even though it isn't statically used directly.